packages/launch/_assets/layout.css

```diff
@@ -362,7 +363,7 @@ li.current > ul > li.anchor {
     position: sticky;
     height: 100%;
     left: 0;
-    top: 70px;
+    top: 141px;
```

Comment:
Makes it behave better when you scroll the page, so the sidebar stays in its place properly. I saw slight movement on Google Chrome at 140px so I made it 141 🙈